pkg/orchestrator/controller/controller.go

```diff
@@ -59,6 +61,8 @@ func New(federator federate.Federator) *controller {
 }
 
 func (c *controller) Start() error {
+	submarinerv1.AddToScheme(scheme.Scheme)
```

Comment:
this should be done in the main. The scheme is supposed to be populated/initialized at the beginning and not modified afterwards.

Reply:
This kind of is part of the beginning, also I thought it should the responsibility of this "class" to do it since it relies on it.  But in main is fine too - as long as it's done somewhere before the   informers start.

Reply:
According to some "documentation" I've seen on the type registry, the preferred way is to add it to the registry before any interaction with the API and for that they usually recommend the main